packages/appstore/src/components/containers/trading-app-card.tsx

```diff
@@ -101,10 +108,16 @@ const TradingAppCard = ({
             <div className={classNames('trading-app-card__container', { 'trading-app-card--divider': has_divider })}>
                 <div className='trading-app-card__details'>
                     <div>
-                        <Text className='title' size='xs' line_height='s' color='prominent'>
-                            {!is_real && sub_title ? `${sub_title} ${localize('Demo')}` : sub_title}
-                        </Text>
-                        {short_code_and_region && (
+                        {is_wallet ? (
+                            <Text className='title' size='xs' line_height='s' color='prominent'>
+                                {is_wallet_demo && sub_title ? `${sub_title} ${localize('Demo')}` : sub_title}
+                            </Text>
+                        ) : (
+                            <Text className='title' size='xs' line_height='s' color='prominent'>
+                                {!is_real && sub_title ? `${sub_title} ${localize('Demo')}` : sub_title}
+                            </Text>
+                        )}
```

Comment:
```suggestion
                        {is_wallet ? (
                            <Text className='title' size='xs' line_height='s' color='prominent'>
                                {is_wallet_demo && sub_title ? `${sub_title} ${localize('Demo')}` : sub_title}
                            </Text>
                        ) : (
                            <Text className='title' size='xs' line_height='s' color='prominent'>
                                {!is_real && sub_title ? `${sub_title} ${localize('Demo')}` : sub_title}
                            </Text>
                        )}
```
There is no difference between these two conditions